modules/core/src/opencl/runtime/generator/parser_clamdblas.py

```diff
@@ -94,15 +94,15 @@
 numEnabled = readFunctionFilter(fns, filterFileName)
 
 functionsFilter = generateFilterNames(fns)
-filter_file = open(filterFileName, 'wb')
+filter_file = open(filterFileName, 'w')
 filter_file.write(functionsFilter)
 
 ctx = {}
 ctx['CLAMDBLAS_REMAP_ORIGIN'] = generateRemapOrigin(fns)
 ctx['CLAMDBLAS_REMAP_DYNAMIC'] = generateRemapDynamic(fns)
 ctx['CLAMDBLAS_FN_DECLARATIONS'] = generateFnDeclaration(fns)
 
-sys.stdout = open('../../../../include/opencv2/core/opencl/runtime/autogenerated/opencl_clamdblas.hpp', 'wb')
+sys.stdout = open('../../../../include/opencv2/core/opencl/runtime/autogenerated/opencl_clamdblas.hpp', 'w')
```

Comment:
Perhaps it makes sense to rename these files too (=> clblas / clfft)

Reply:
I have now pushed changes to rename *clamdblas* files to *clblas*, and *clamdfft* files to *clff*.